src/Components/Body.tsx

```diff
@@ -225,118 +302,294 @@ export function Body(props: State) {
-              maxHeight: "200px",
-              minWidth: "300px",
-              minHeight: "200px",
+              alignItems: "center",
+              display: "flex",
+              flexDirection: "column",
+              width: "400px",
             }}
           >
-            {[...Array(7).keys()].map((item) => (
-              <Grid item>
+            <Box className="box-header" mt={2} mb={2}>
+              <Box mt={2}>
                 <Button
-                  disableRipple={true}
-                  color="secondary"
-                  size="small"
-                  variant="text"
+                  onClick={() => updateTransition(0)}
                   style={{
                     maxWidth: "30px",
                     maxHeight: "30px",
                     minWidth: "30px",
                     minHeight: "30px",
                   }}
+                  size="small"
+                  variant="text"
                 >
-                  {day[item]}
+                  <ArrowForwardIosIcon />
                 </Button>
-              </Grid>
-            ))}
-            {[...Array(props.daysInMonth[props.index]).keys()].map((item) => (
-              <Grid key={item} item>
+              </Box>
+              <Box mt={2} mr={1}>
+                <Typography color="textPrimary" variant="h5">
+                  {props.dateRange.dates[props.index].toLocaleString(
+                    "default",
+                    {
+                      month: "long",
+                    }
+                  )}
+                </Typography>
+              </Box>
+              <Box mt={2}>
+                <Typography color="secondary" variant="h5">
+                  {props.dateRange.dates[props.index].getFullYear()}
+                </Typography>
+              </Box>
+              <Box ml={1} mt={2}>
                 <Button
-                  onClick={() => handleClick("day", item + 1)}
+                  onClick={() => updateTransition(1)}
                   style={{
                     maxWidth: "30px",
                     maxHeight: "30px",
                     minWidth: "30px",
                     minHeight: "30px",
                   }}
                   size="small"
-                  color="primary"
-                  variant={getVariant(item + 1)}
-                  disableRipple
+                  color="secondary"
+                  variant="text"
                 >
-                  {item + 1}
+                  <ArrowBackIosIcon />
                 </Button>
+              </Box>
+            </Box>
+            <Box ml={4}>
+              <Grid
+                container
+                justify="flex-start"
+                spacing={1}
+                style={{
+                  maxWidth: "300px",
+                  maxHeight: "200px",
+                  minWidth: "300px",
+                  minHeight: "200px",
+                }}
+              >
+                {[...Array(7).keys()].map((item) => (

```

Comment:
I bet there's a nicer calendar rendering widget out there you might be able to leverage

Reply:
I kinda liked mine... Do you want me to look for another one?